tests/test_write.py

```diff
@@ -1,54 +1,18 @@
-import io
-
-import numpy as np
 import pytest
 
-import _roffio.writing as roffwrite
+import roffio
+from _roffio.exceptions import RoffWriteError
+
+
+def test_write_disallows_double_quotation_marks(tmpdir):
+    contents = {"tagname": [("keyname", '"')]}
```

Comment:
Im reading this as in roffio does not support `"` at all, and should rather use `'` instead.
But did you look at matched `"` ?

Reply:
I think this is fine. 👍